use-timescale/continuous-aggregates/compression-on-continuous-aggregates.md

```diff
@@ -5,83 +5,75 @@ products: [cloud, mst, self_hosted]
 
-Continuous aggregates are often used to downsample historical data. If the data
-is only used for analytical queries and never modified, you can compress the
-aggregate to save on storage.
+# Convert continuous aggregates to the columnstore
 
-<Highlight type="warning">
-Before version
-[2.11.0](/about/latest/release-notes/#timescaledb-2110-on-2023-05-22), you can't
-refresh the compressed regions of a continuous aggregate. To avoid conflicts
-between compression and refresh, make sure you set `compress_after` to a larger
-interval than the `start_offset` of your [refresh
-policy](/api/latest/continuous-aggregates/add_continuous_aggregate_policy).
-</Highlight>
+To save on storage costs, you use $HYPERCORE to downsample historical data stored in $CAGGs. After you 
+[enable columnstore][compression_continuous-aggregate] on a `MATERIALIZED VIEW`, you set a 
+[columnstore policy][add_columnstore_policy]. This policy defines the intervals when chunks in a $CAGG
+are compressed as they are converted from the $ROWSTORE to the $COLUMNSTORE.
 
-Compression on continuous aggregates works similarly to [compression on
-hypertables][compression]. When compression is enabled and no other options are
-provided, the `segment_by` value will be automatically set to the group by
-columns of the continuous aggregate and the `time_bucket` column will be used as
-the `order_by` column in the compression configuration.
+$COLUMNSTORE_CAP works in the same way on [$HYPERTABLEs and $CAGGs][hypercore]. When you enable
+$COLUMNSTORE with no other options, your data is [segmented by][alter_materialized_view_arguments] the `groupby` columns 
+in the $CAGG, and [ordered by][alter_materialized_view_arguments] the time column. [Real-time aggregation][real-time-aggregates]
+is disabled by default.
 
-## Enable compression on continuous aggregates
+<Since2200 /> For the old API, see <a href="https://docs.timescale.com/use-timescale/latest/compression/compression-on-continuous-aggregates/">Compress continuous aggregates</a>.
 
-You can enable and disable compression on continuous aggregates by setting the
-`compress` parameter when you alter the view.
+## Configure $COLUMNSTORE on $CAGGs
+
+For an [existing $CAGG][create-cagg]:
 
 <Procedure>
 
-### Enabling and disabling compression on continuous aggregates
+1. **Enable $COLUMNSTORE on a $CAGG**
 
-1.  For an existing continuous aggregate, at the `psql` prompt, enable
-    compression:
+   To enable the $COLUMNSTORE compression on a $CAGG, set `timescaledb.enable_columnstore = true` when you alter the view:
 
-    ```sql
-    ALTER MATERIALIZED VIEW cagg_name set (timescaledb.compress = true);
-    ```
+   ```sql
+   ALTER MATERIALIZED VIEW <cagg_name> set (timescaledb.enable_columnstore = true,);
+   ```
+   To disable the $COLUMNSTORE compression, set  `timescaledb.enable_columnstore = false`:
 
-1.  Disable compression:
+   IAIN: there was a note here saying this does not work on CAGGS with compressed chunks
```

Comment:
Hi Iain 👋 

compressed chunks work with CAggs, but not flawlessly. We have a know issue, that locking issues can happen between the refresh policy and compression policy fighting for locks.

Reply:
Ok, I've removed the note. The locking issue is in the release notes?

Reply:
The potential locking issue is present since multiple releases 😓